Wox/SettingWindow.xaml

```diff
@@ -33,6 +33,9 @@
     <TabControl Height="auto" SelectedIndex="0">
         <TabItem Header="{DynamicResource general}">
             <StackPanel Orientation="Vertical">
+                <CheckBox Margin="10" IsChecked="{Binding PortableMode}">
+                    <TextBlock Text="Portable Mode" />
```

Comment:
maybe add a tooltip explaining what is portable mode?

Reply:
Probably at this stage don't worry about a tooltip, later on we can comeback to do a tooltip for all of them? Ideally it should be self explanatory enough to not require one.